src/Sculpin/Contrib/ProxySourceCollection/ProxySourceCollectionDataProvider.php

```diff
@@ -37,6 +37,7 @@ public function __construct(
         FormatterManager $formatterManager,
         $dataProviderName,
         $dataSingularName = null,
+        // FIXME violates PSR2; defaults at end of method
```

Comment:
Make `$filter` and `$map` both default to `null`. This should fix PSR-2 complaint.